### ISSUE TITLE
hemera-ajv doesnt seem to be working correctly

### ISSUE THREAD

**Issue by BransonGitomeh** [hemerajs/hemera#233]
Before you submit an issue we recommend you drop into the [Gitter community](https://gitter.im/hemerajs/hemera) and ask any questions you have or mention any problems you've had getting started with hemera.

## Description
Hemera ajv doesnt seem to be validating as expected

## Steps to Reproduce
Example:
1. Start NATS
2. Use plugin in test script as follows
```
const Hemera = require('nats-hemera')
const nats = require('nats').connect()

const hemera = new Hemera(nats, {
    logLevel: 'info'
})

hemera.ready(() => {
    hemera.add(
        {
            topic: 'math',
            cmd: 'add',
            schema: {
                request: {
                    properties: {
                        a: { type: 'number' },
                        b: { type: 'number' }
                    },
                    required: ['c']
                },
                response: {
                    properties: {
                        a: { type: 'number' },
                        b: { type: 'number' }
                    },
                    required: ['c']
                }
            }
        },
        (req, cb) => {
            cb(null, req.a + req.b)
        }
    )

    hemera.act(
        {
            topic: 'math',
            cmd: 'add',
            a: 1,
            b: 2
        },
        function (err, resp) {
            if (err) {
                console.log(err)
            }

            console.log(resp, '<-------------------Result')
        }
    )
})
```
3. Send message
4. Error appears in the logs

## Expected Result
a validation error will be thrown since "c" is missing in the action call

## Actual Result
no error is thrown, execution of the action occurs as expected

## Context

- [ ] nats-hemera (core)
- [ ] hemera-zipkin
- [ ] hemera-store
- [ ] hemera-stats
- [ ] hemera-rabbitmq
- [ ] hemera-nsq
- [ ] hemera-elasticsearch
- [ ] hemera-joi
- [ ] hemera-msgpack
- [ ] hemera-avro
- [ ] hemera-jwt-auth
- [x] hemera-ajv

## Your Environment
- NATS: nats-server version 2.0.0-alpha
- NODE: 8.9.4
```
    "hemera": "^0.0.5",
    "hemera-ajv": "^0.2.5",
```
- Environment name and version: 
- Operating System and version: linux manjaro

**Comment by StarpTech** [comment]
Hi @BransonGitomeh thank you for the bug report `ajv` doesn't mark the payload as invalid if you pass a primitive (`undefined`, `true`) to the validation function but expect an object based on the schema. This case has to be handled. Would you like to send a PR?

**Comment by BransonGitomeh** [comment]
hey i went with doing the integration manually with ajv. would be open to test this new solution. thanks! sorry i dont have the capacity to make a PR

**Comment by StarpTech** [comment]
Hi @BransonGitomeh I already fixed it. It will be released in some minutes.

**Comment by StarpTech** [comment]
It was released with hemera-ajv@0.2.6
